### PR TITLE
stable/kubernetes-dashboard

### PR DESCRIPTION
**Pull request by poochwashere**
#### What this PR does / why we need it:
* Fixed a spelling error in `readme`
* Fixed broken link for `RBAC` definitions. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.9.0
+version: 1.9.1
```
Comment (kfox1111) [inline stable/kubernetes-dashboard/Chart.yaml:3]: 1.10.1. Otherwise looks good.

Comment (poochwashere) [reply stable/kubernetes-dashboard/Chart.yaml:3]: I updated the version number and pushed the change.

**Comment by zanhsieh** [comment]
@poochwashere 
Please sign again for DCO and resolve stable/kubernetes-dashboard/Chart.yaml conflict. Thanks for your contribution.

**Comment by poochwashere** [comment]
@zanhsieh I signed the DCO and pushed with the signature, I am not sure why this is still failing. I am green at code contributions....

File: stable/kubernetes-dashboard/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.9.0
+version: 1.10.1
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/Chart.yaml]:
```suggestion
version: 1.9.1
```

**Comment by zanhsieh** [comment]
@poochwashere 
This is what DCO report shows:
> Commit` sha: c003e7e, Author: Jeff Puccinelli, Committer: Jeff Puccinelli; The sign-off is missing.

Also for e2e test failure, you probably need to follow @davidkarlsen suggested to bump the chart version first and then run '/test pull-charts-e2e' (don't run '/retest') it somehow broken.

**Pull request closed without merging**